k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-configmap/templates/_config.tpl

```diff
@@ -30,53 +29,29 @@ cortx:
       device_certificate: /etc/cortx/solution/ssl/stx.pem
   utils:
     message_bus_backend: kafka
-  s3:                                                               # DEPRECATED - ENTIRE S3 KEY
-    iam:
-      endpoints:
-      - https://{{ $ioSvcName }}:9443
-      - http://{{ $ioSvcName }}:9080
-    data:
-      endpoints:
-      - http://{{ $ioSvcName }}:80
-      - https://{{ $ioSvcName }}:443
-    internal:
-      endpoints:
-      - http://{{ $ioSvcName }}:28049
-    {{- with .Values.cortxS3.instanceCount }}
-    service_instances: {{ . | int }}
-    {{- end }}
-    io_max_units: 8
-    {{- with .Values.cortxS3.maxStartTimeout }}
-    max_start_timeout: {{ . | int }}
-    {{- end }}
+  rgw:
+    {{- $ioSvcName := required "A valid cortxIoService.name is required!" .Values.cortxIoService.name }}
+    {{- $iosvcHttpPort := required "A valid cortxIoService.ports.http is required!" .Values.cortxIoService.ports.http }}
+    {{- $iosvcHttpsPort := required "A valid cortxIoService.ports.https is required!" .Values.cortxIoService.ports.https }}
```

Comment:
This could be achieved by a Helm `lookup`, however the deployment script installs this chart prior to the cortx-platform chart, so the service is not available.